File: src/main/java/mods/orca/mffs/client/MFFSTab.java
package mods.orca.mffs.client;

import mods.orca.mffs.MFFSMod;
import mods.orca.mffs.items.ModItems;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class MFFSTab extends CreativeTabs {

    public static final MFFSTab mffsTab = new MFFSTab();

    public MFFSTab() {
        super(MFFSMod.MODID);
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(ModItems.FREQCARD_BLANK);
    }
}

File: src/main/java/mods/orca/mffs/client/MFFSTab.kt
package mods.orca.mffs.client

import mods.orca.mffs.MFFSMod
import mods.orca.mffs.blocks.core.BlockForceFieldCore
import mods.orca.mffs.items.ItemFrequencyCard
import mods.orca.mffs.items.ItemFrequencyCardBlank
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

/**
 * The creative tab used for our mod items.
 */
object MFFSTab : CreativeTabs(MFFSMod.modId) {
    fun setup() {
        BlockForceFieldCore.creativeTab = this
        ItemFrequencyCardBlank.creativeTab = this
    }

    override fun createIcon(): ItemStack {
        return ItemStack(ItemFrequencyCard)
    }
}
